interfaces/apparmor/backend.go

```diff
@@ -855,6 +843,30 @@ func (b *Backend) addContent(securityTag string, snapInfo *snap.Info, cmdName st
 			return templateVariables(snapInfo, securityTag, cmdName)
 		case "###PROFILEATTACH###":
 			return fmt.Sprintf("profile \"%s\"", securityTag)
+		case "###FLAGS###":
+			// default flags
+			flags := []string{"attach_disconnected", "mediate_deleted"}
+			if spec.Unconfined() {
+				// need both parser and kernel support for unconfined
+				pfeatures, _ := parserFeatures()
+				kfeatures, _ := kernelFeatures()
+				if strutil.ListContains(pfeatures, "unconfined") &&
+					strutil.ListContains(kfeatures, "policy:unconfined_restrictions") {
+					flags = append(flags, "unconfined")
```

Comment:
what's the effect on the labeling of the profile of this flag? none?

Reply:
The effect on the label is none. But it does reflect in the context (what you get out /proc/<pid>/attr/current). It will show up as the mode there. Eg.

```
  snap.chromium.chromium (unconfined)
```

where for something that is confined and enforcing you would get
```
  snap.chromium.chromium (enforce)
```

Reply:
As for tying it to an interface, I don't think that is a bad idea, the easier it is to track where these are the better.